outputs.tf

```
@@ -1,40 +1,48 @@
-output "alb-sg-id" {
-  value = aws_security_group.ecs-alb-sg.id
+output "alb_sg_id" {
+  description = "Resource ID for Security Group applied to ALB"
+  value       = aws_security_group.ecs_alb_sg.id
 }
 
-output "ec2-sg-id" {
-  value = aws_security_group.ecs_ec2_sg.id
+output "ec2_sg_id" {
+  description = "Resource ID for Security Group applied to EC2 instances"
+  value       = aws_security_group.ecs_ec2_sg.id
 }
 
-output "cluster-id" {
-  value = aws_ecs_cluster.ecs-cluster.id
+output "cluster_id" {
+  description = "Resource ID of ECS cluster"
+  value       = aws_ecs_cluster.ecs_cluster.id
 }
 
-output "alb-arn" {
-  value = aws_alb.ecs-load-balancer.arn
+output "alb_arn" {
+  description = "ARN of ALB"
+  value       = aws_alb.ecs_load_balancer.arn
 }
 
-output "alb-arn-suffix" {
-  value = aws_alb.ecs-load-balancer.arn_suffix
+output "alb_arn_suffix" {
+  description = "ARN suffix of ALB"
+  value       = aws_alb.ecs_load_balancer.arn_suffix
 }
 
-output "alb-dns" {
-  value = aws_alb.ecs-load-balancer.dns_name
+output "alb_dns_name" {
+  description = "DNS name of ALB"
+  value       = aws_alb.ecs_load_balancer.dns_name
 }
 
-output "alb-zone" {
-  value = aws_alb.ecs-load-balancer.zone_id
+output "alb_zone_id" {
+  description = "R53 zone ID of ALB"
+  value       = aws_alb.ecs_load_balancer.zone_id
 }
 
-output "cluster-name" {
-  value = aws_ecs_cluster.ecs-cluster.name
+output "cluster_name" {
```

```suggestion
output "ecs_cluster_name" {
```

also some mad sorting issues in here